src/signer.ts

```diff
@@ -1,6 +1,7 @@
 import sha256 from 'js-sha256';
 import { Signature, KeyPair, PublicKey } from './utils/key_pair';
-import { KeyStore, InMemoryKeyStore } from './key_stores';
```

Comment:
What is the reason to do this?

Reply:
This bug https://github.com/near/near-api-js/issues/397

Apparently we pack doesn’t care that only specific classes are imported, key stores module ends up depending on fs.